apps/frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -77,6 +81,14 @@ const Checkbox = <T extends FieldValues, OptionValueT extends number | string>({
 			>
 				{label}
 			</p>
+			<i
+				className={getValidClassNames(
+					styles["tip"],
+					!hasTip && "visually-hidden",
```

Comment:
If there's no tip, then don't display it at all. Currently you are hiding an empty thing and it doesn't make sense